src/metabase/driver/common/parameters/values.clj

```diff
@@ -208,27 +208,28 @@
   converted to SQL as a simple comma-separated list.)"
   [value]
   (cond
-   ;; if not a string it's already been parsed
-   (number? value) value
-   ;; same goes for an instance of CommaSeperated values
-   (instance? CommaSeparatedNumbers value) value
-
-   ;; newer operators use vectors as their arguments even if there's only one
-   (vector? value)
-   (let [values (map parse-number value)]
-     (if (next values)
-       (i/map->CommaSeparatedNumbers {:numbers values})
-       (first values)))
-   ;; if the value is a string, then split it by commas in the string. Usually there should be none.
-   ;; Parse each part as a number.
-   (string? value)
-   (let [parts (for [part (str/split value #",")]
-                 (parse-number part))]
-     (if (> (count parts) 1)
-       ;; If there's more than one number return an instance of `CommaSeparatedNumbers`
-       (i/map->CommaSeparatedNumbers {:numbers parts})
-       ;; otherwise just return the single number
-       (first parts)))))
+    ;; if not a string it's already been parsed
+    (number? value)
+    value
+    ;; same goes for an instance of CommaSeperated values
+    (instance? CommaSeparatedNumbers value)
+    value
+    ;; newer operators use vectors as their arguments even if there's only one
+    (vector? value)
+    (let [values (mapv value->number value)]
+      (if (next values)
+        (i/map->CommaSeparatedNumbers {:numbers values})
+        (first values)))
+    ;; if the value is a string, then split it by commas in the string. Usually there should be none.
+    ;; Parse each part as a number.
+    (string? value)
+    (let [parts (for [part (str/split value #",")]
+                  (parse-number part))]
+      (if (> (count parts) 1)
+        ;; If there's more than one number return an instance of `CommaSeparatedNumbers`
+        (i/map->CommaSeparatedNumbers {:numbers parts})
+        ;; otherwise just return the single number
+        (first parts)))))
```

Comment:
Unrelated to fix -- fixed the indentation here (`cond` should have its body indented two spaces)